src/App.tsx

```diff
@@ -3,12 +3,17 @@ import './App.css';
 import {NavigationContextContextProvider} from "./Contexts/NavigationContext";
 import {HomePage} from "./Pages/HomePage";
 import {RoutesEnum} from "./Routes";
+<<<<<<< Updated upstream
 import {MemoryRouter, Route, Routes} from "react-router-dom";
+=======
+import Login from "./Login/Login";
+import {HashRouter, MemoryRouter, Route, Routes} from "react-router-dom";
+>>>>>>> Stashed changes
```

Comment:
some funny things are happening here due to git commits

Reply:
<img width="809" alt="image" src="https://user-images.githubusercontent.com/126940746/224474312-93b2ec04-d261-4bcd-b1c2-499d06b4685a.png">